config-migration-plan.md

```
@@ -0,0 +1,213 @@
+# CivicTheme Configuration Migration Plan - 1.10 → 1.12
+
+## Objective
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)